app/controllers/TaxYearOverviewController.scala

```diff
@@ -19,25 +19,29 @@ package controllers
 import audit.AuditingService
 import audit.models.TaxYearOverviewResponseAuditModel
 import auth.MtdItUser
-import config.featureswitch.{CodingOut, FeatureSwitching, TxmEventsApproved}
```

Comment:
are we removing txmeventsapproved feature switch?

Reply:
Yes, these are actually added at the time of Release 6. So whereever possible we could remove it.